controllers/workoutlog.js

```diff
@@ -0,0 +1,10 @@
+const log = require('../models/workoutlog')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const log = require("../models/workoutlog")
```